gilbreth_perception/config/parameters.yaml

```diff
@@ -5,7 +5,7 @@ parameters:
   max_correspondence_distance: 0.01
   nr_iterations: 500
 switches:
-  visualizer: true
+  visualizer: false
```

Comment:
I don't think that the visualizer is needed as you can always look at the point cloud in rviz, however lets not address that right now.